src/components/screens/ExperimentCreator.vue

```diff
@@ -0,0 +1,1348 @@
+                          @add="acceptNewDoiIdentifier"
+                          @keyup.escape="clearDoiIdentifierSearch"
+                        >
+                          <template #chip="slotProps">
+                            <div>
+                                <span>{{ slotProps.value.identifier }}</span>
+                            </div>
+                        </template>
+                      </Chips>
+                      <label :for="$scopedId('input-doiIdentifiers')">DOIs</label>
+                    </span>
+                    <span v-if="validationErrors.doiIdentifiers" class="mave-field-error">{{validationErrors.doiIdentifiers}}</span>
+                  </div>
+                </div>
+                <div v-if="itemStatus == 'NotLoaded' || this.item.private == true">
+                  <div class="mavedb-wizard-row">
+                    <div class="mavedb-wizard-help">
+                      <label>
+                        Contributors who may edit this experiment. Enter each contributor's
+                        <a href="https://orcid.org" target="_blank">ORCID</a> ID and confirm their name.
+                      </label>
+                      <div class="mavedb-help-small">
+                        Examples: 1111-1111-1111-1111
+                      </div>
+                    </div>
+                    <div class="mavedb-wizard-content field">
+                      <span class="p-float-label">
+                        <Chips
+                          ref="contributorsInput"
+                          v-model="contributors"
+                          :id="$scopedId('input-contributors')"
+                          :addOnBlur="true"
+                          :allowDuplicate="false"
+                          placeholder="Type or paste ORCID IDs here."
+                          @add="newContributorsAdded"
+                          @keyup.escape="clearContributorSearch"
+                        >
+                          <template #chip="slotProps">
+                            <div>
+                              <div v-if="slotProps.value.givenName || slotProps.value.familyName">{{ slotProps.value.givenName }} {{ slotProps.value.familyName }} ({{ slotProps.value.orcidId }})</div>
+                              <div v-else>{{ slotProps.value.orcidId }}</div>
+                            </div>
+                          </template>
+                        </Chips>
+                        <label :for="$scopedId('input-contributors')">Contributors</label>
+                      </span>
+                      <span v-if="validationErrors.contributors" class="mave-field-error">{{validationErrors.contributors}}</span>
+                    </div>
+                  </div>
+                  <div class="mavedb-wizard-row">
+                    <div class="mavedb-wizard-help">
+                      <label>
+                        Any publications associated with the experiment.
```

Comment:
Let's make the help text for publication identifiers the same on this page as already exists on the score set page.

Reply:
Both of them are nearly the same, but this one is for experiment so I changed the "score set" to "experiment" in the help text. I think it's better to say "Any publications associated with the experiment" rather than "Any publications associated with the score set" in creating new experiment page. The additional text is "Publications included in an experiment will also be displayed on their associated score set pages". It's from our document https://mavedb.org/docs/mavedb/upload_guide.html. I think it's helpful for users to understand what they can put in experiment so I added.